src/compiler/program.ts

```diff
@@ -79,9 +79,7 @@ namespace ts {
             return false;
         }
 
-        const i = moduleName.lastIndexOf("./", 1);
-        const startsWithDotSlashOrDotDotSlash = i === 0 || (i === 1 && moduleName.charCodeAt(0) === CharacterCodes.dot);
-        return !startsWithDotSlashOrDotDotSlash;
+        return !startsWithDotSlashOrDotDotSlash.test(moduleName);

```

Comment:
The idea here is that looking up the fist 2 chars is cheeper than running a regexp. is there a reason why we need to change it?